app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CanTheyUseOurServiceController.scala

```diff
@@ -378,9 +399,10 @@ object CanTheyUseOurServiceController {
 
   def disposalDateForm(maximumDateInclusive: LocalDate): Form[DisposalDate] = Form(
     mapping(
-      "date" -> of(
+      "" -> of(
```

Comment:
how can this be an empty string?

Reply:
the actual mapping is inside the formatter, so it doesn't matter what this particular string is